package-lock.json

```diff
@@ -7,7 +7,7 @@
     "@babel/code-frame": {
```

Comment:
kann man die Datei in gitignore aufnehmen? Das ist immer bei den unstaged changes dabei.